velox/dwio/dwrf/proto/CMakeLists.txt

```diff
@@ -30,12 +30,15 @@ endforeach()
 set(PROTO_OUTPUT_FILES ${PROTO_HDRS} ${PROTO_SRCS})
 set_source_files_properties(${PROTO_OUTPUT_FILES} PROPERTIES GENERATED TRUE)
 
+foreach(PROTO_PATH ${CMAKE_SOURCE_DIR} ${Protobuf_INCLUDE_DIRS})
+  list(APPEND PROTO_PATH_ARGS --proto_path=${PROTO_PATH})
+endforeach()
```

Comment:
Why is this needed vs. the previous version where it's hardcoded in the  `COMMAND`?  Is `PROTO_PATH_ARGS` generated by the protobuf config or something? Could you add a comment?

Reply:
For some weird reason, with the previous line...
  --proto_path ${Protobuf_INCLUDE_DIRS} --cpp_out ${PROJECT_SOURCE_DIR}
...the MacOS builds failed with "--proto_path without argument".
This because the variable Protobuf_INCLUDE_DIRS is empty when Protobuf is installed in the system.
So I was forced to make sure that --proto_path is passed an argument.

Reply:
Let's add a comment for future reference.

Reply:
Agree. I added comments.